src/pages/index.astro

```diff
@@ -3,192 +3,152 @@ import Main from '../layouts/Main.astro'
+	width: 65%;
 }
 
 h1 {
-    font-family: var(--mainfont);
-    text-transform: uppercase;
-    font-weight: 200;
-    font-size: 50px;
-    letter-spacing: 4px;
-}
-#hero p {
-    width: 75%;
-    font-weight: 100;
-    font-size: 14pt;
+	font-weight: 200;
+	font-size: 3.2rem;
+	letter-spacing: 0.08em;
+	text-transform: uppercase;
 }
 
-.center-title {
-    text-align: center;
+#hero p {
+	width: 75%;
+	font-weight: 100;
+	font-size: 1.2rem;
 }
 
 h2 {
-    font-size: 30pt
-}
-
-.center-title {
-    text-align: center;
-}
-
-#pillars {
-    display: flex;
-    justify-content: space-evenly;
-    margin-bottom: 5rem;
-    z-index: -9;
-    -webkit-transform: translateZ(-1);
-    transform: translateZ(-1);
-}
-
-.pillar-box {
-    font-size: 18pt;
-    margin-top: 50px;
-    text-align: center;
-    padding: 10px 20px;
-    color: black;
-    font-weight: bold;
-    border-radius: 2px;
+	font-size: 2.3rem;
+	text-align: center;
 }
 
-.pillar {
-    flex-basis: 100%;
-    margin: 0 25px;
-    max-width: 400px;
-    clip-path: polygon(0 7%, 100% 0, 100% 93%, 0 100%);
-    padding: 2.5rem 2.8rem;
-    min-height: 400px;
+.pillars {
+	display: flex;
+	gap: 25px;
+	padding: 25px;
+	justify-content: space-evenly;
+	margin-bottom: 5rem;
+	z-index: -9;
+	transform: translateZ(-1);
 }
 
-#pillar1 {
-    background: linear-gradient(to right, #f783a7, #fcd5e1);
+.pillars section
+{
+	max-width: 400px;
+	clip-path: polygon(0 7%, 100% 0, 100% 93%, 0 100%);
+	padding: 3rem 2.8rem;
+	min-height: 400px;
+	color: var(--black);
 }
 
-#pillar2 {
-    background: linear-gradient(to right, #fcd5e1, #fff, #def8f8);
-    margin-top: 50px;
-}
-#pillar3 {
-    background: linear-gradient(to right, #def8f8, #9eecec);
-    margin-top: 100px;
+.pillars section:nth-child(1) {
+	background: linear-gradient(to right, #f783a7, #fcd5e1);
 }
 
-.pillar p {
-    padding: .5em 0;
-    text-align: left;
-    color: black;
-    font-size: 13pt;
-    line-height: 27px;
+.pillars section:nth-child(2) {
+	background: linear-gradient(to right, #fcd5e1, #fff, #def8f8);
+	margin-top: 50px;
 }
-
-#work-title {
-    margin-top: 130px;
+.pillars section:nth-child(3){
+	background: linear-gradient(to right, #def8f8, #9eecec);
+	margin-top: 100px;
 }
 
-#button-continer {
-    position: relative;
+.pillars h3 {
+	font-size: 1.3rem;
+	text-align: center;
+	margin: 2rem 0;
 }
 
-@media screen and (max-width: 1290px) {
-    .pillar {
-        margin: 0 12px;
-    }
-
+.pillars p {
+	font-size: 13pt;
+	line-height: 27px;
 }
 
 @media screen and (max-width: 1050px) {
-    .pillar {
-        max-width: unset;
-        margin: 50px 0;
-        padding: 2.5rem 1.2rem;
-        margin-bottom: 50px;
-    }
-
-    #pillars {
-        display: unset;
-    }
-
-}
-
-@media screen and (max-width: 1050px) {
-    #pillars {
-        display: unset;
-    }
+	.pillars {
+		display: block;
+	}
+	
+	.pillars section {
+		max-width: unset;
+		min-height: unset;
+		margin-top: 5px !important;
```

Comment:
Apologies, but this is required now that the individual pillars are selected much more strongly above (for the individual `margin-top` values for each descending pillar).